scripts/split-e2e-tests-codebuild.ts

```diff
@@ -342,6 +323,20 @@ function main(): void {
     },
   );
   let allBuilds = [...splitE2ETests, ...splitMigrationV5Tests, ...splitMigrationV6Tests, ...splitMigrationV10Tests];
+  const dependeeIdentifiers: string[] = allBuilds.map((buildObject) => buildObject.identifier);
```

Comment:
Does the array order matter here? If not, I prefer if it was sorted for readability

```suggestion
  const dependeeIdentifiers: string[] = allBuilds.map((buildObject) => buildObject.identifier).sort();
```